AppServer/google/appengine/api/appcontroller_client.py

```diff
@@ -329,6 +329,19 @@ def add_routing_for_appserver(self, app_id, appserver_ip, port):
       app_id, appserver_ip, port, self.secret)
 
 
+  def remove_routing_for_appserver(self, version_key, ip, port):
```

Comment:
This is so that the AppManager can make the call. We don't have a separate AC client for services and the dashboard.